user-guide/Troubleshooting/Troubleshooting_Flowcharts/Troubleshooting_Startup_Issues.md

```diff
@@ -4,39 +4,258 @@ uid: Troubleshooting_Startup_Issues
+click NATSIssue "/user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.html#nats-issue"
+click CassandraIssue "/user-guide/Troubleshooting/Procedures/Troubleshooting_SLScripting_Issues.html#cassandra-issue"
+click ContactTechSupport "/user-guide/Troubleshooting/Contacting_tech_support.html"
 %% Apply styles to blocks %%
-class START classTerminal;
-class CHECK1 classDecision;
-class HOME,licensing,nicpriority,dbOffloading,failingHeartbeats,others classExternalRef;
+class DMAStartupIssues,ContactTechSupport DarkBlue;
+class LicensingIssue,IPChangeIssue,NATSIssue,CassandraIssue classExternalRef;
+class AgentNotLicensed,SLNetComFailure,IncorrectIPAddresses,NATSErrors,CassandraErrors Blue;
+class Home LightBlue;
 ```
+
+## Licensing issue
+
+### Symptoms
+
+- DataMiner fails to start, takes too long to start, or gets stuck at 99% during the startup process.
+
+- DataMiner Cube displays the error message "This DataMiner Agent is not licensed", prompting you to contact the system administrator.
+
+- Errors in the *SLDataMiner.txt* file, including:
+
+  - No valid license file
+  - Start DataMiner failed, 0x80040291
+
+### Root cause
+
+DataMiner cannot find a valid license to run the Agent. This issue may be caused by a **MAC address change**, which can have the following causes:
+
+- Hardware changes (e.g. replacing network interface cards).
+
+- NIC priority rearrangement (e.g. because of new VMs or network configurations).
+
+- Randomized MAC addresses.
+
+### Solution
+
+1. Navigate to the folder `C:\Skyline DataMiner` on the DMA.
+
+1. Copy the *Request.lic* file and send it to <dataminer.licensing@skyline.be>.
+
+   After some time, you will receive an email with license files.
+
+1. When you have received the license files, place them in the `C:\Skyline DataMiner` folder.
+
+1. Restart the DataMiner Agent.
+
+## IP change issue
+
+Two different DataMiner startup issues are related to IP changes: DataMiner may be trying to [connect to an old IP address](#dataminer-is-trying-to-connect-to-an-old-ip-address), or there may be [incorrect IP addresses in *DMS.xml*](#incorrect-ip-addresses-in-dmsxml-cause-ip-conflicts).
+
+### DataMiner is trying to connect to an old IP address
+
+#### Symptoms
+
+- DataMiner fails to start or gets stuck at 99%.
+
+- Error messages in *SLDataMiner.txt* indicate a failure to initialize SLNetCom.
+
+  For example:
+
+  ```txt
+  Initializing SLNetCom
+  Initializing SLNetCom failed. - There's no connection available with this dataminer. (hr = 0x800402CD)
+  ```
+
+- Authentication timeouts are indicated in *SLNet.txt*.
+
+  For example:
+
+  - `Destroying connection (SLAnalytics): Authentication took too long.`
+  - `Destroying connection (SLDataMiner.exe): Authentication took too long.`
+
+- NATS logs indicate "connected to NATS for account and activation notifications" and "STREAM: Streaming Server is ready".
```

Comment:
Where can these NATS logs be found? Could we add an exact example of such a log entry?

Reply:
This screenshot appears to refer to the lines above, rather than the line about the NATS logs. I assume this confirms that indeed, those are included in SLNet.txt, but it means that my question for line 119 remains unanswered.